FL_Studio/FLStudio.munki.recipe

```diff
@@ -88,12 +124,25 @@
             <key>Processor</key>
             <string>MunkiImporter</string>
         </dict>
+        <dict>
+             <key>Arguments</key>
+             <dict>
+                 <key>mode</key>
+                 <string>777</string>
+                 <key>resource_path</key>
+                 <string>%RECIPE_CACHE_DIR%/%NAME%</string>
+             </dict>
+             <key>Processor</key>
+             <string>com.github.grahampugh.recipes.commonprocessors/ChangeModeOwner</string>
```

Comment:
I have no interest in including a non-standard autopkg processor as a requirement in the recipe flow when it's not necessary to do so.

Please use one of the many other standard processors (like [PkgRootCreator](https://github.com/autopkg/autopkg/wiki/Processor-PkgRootCreator)) that are sufficient for creating a directory with a desired set of permission.